dask/array/routines.py

```diff
@@ -515,34 +515,42 @@ def gradient(f, *varargs, **kwargs):
     return results
 
 
+def _bincount_sum(bincounts, dtype=int):
+    n = max(map(len, bincounts))
+    out = np.zeros(n, dtype=dtype)
+    for b in bincounts:
+        out[:len(b)] += b[:len(b)]
+    return out
+
+
 @wraps(np.bincount)
-def bincount(x, weights=None, minlength=None):
-    if minlength is None:
-        raise TypeError("Must specify minlength argument in da.bincount")
+def bincount(x, weights=None, minlength=0):
     assert x.ndim == 1
     if weights is not None:
         assert weights.chunks == x.chunks
```

Comment:
Admittedly this code precedes this PR. That said, it would be nice to have something other than `assert`s here. If you don't feel like tackling that in this PR, we can raise an issue. Am just noting this for now so we don't forget 🙂

Reply:
Raise a value error, maybe?